File: python/Elektronik.py
from Produk import Produk

class Elektronik(Produk):
    def __init__(self, nama = "", harga = "", merek = "", garansi = "", baterai = "", warna = ""):
        super().__init__(nama, harga, merek)
        self.__garansi = garansi
        self.__baterai = baterai
        self.__warna = warna

    def getGaransi(self):
        return self.__garansi

    def setGaransi(self, garansi):
        self.__garansi = garansi

    def getBaterai(self):
        return self.__baterai

    def setBaterai(self, baterai):
        self.__baterai = baterai

    def getWarna(self):
        return self.__warna

    def setWarna(self, warna):
        self.__warna = warna
        

File: java/Elektronik.java
class Elektronik extends Produk {
    private String garansi;
    private String baterai;
    private String warna;

    public Elektronik() {
        super();
        this.garansi = "";
        this.baterai = "";
        this.warna = "";
    }

    public Elektronik(String nama, String harga, String merek, String garansi, String baterai, String warna) {
        super(nama, harga, merek);
        this.garansi = garansi;
        this.baterai = baterai;
        this.warna = warna;
    }

    public String getGaransi() {
        return garansi;
    }

    public void setGaransi(String garansi) {
        this.garansi = garansi;
    }

    public String getBaterai() {
        return baterai;
    }

    public void setBaterai(String baterai) {
        this.baterai = baterai;
    }

    public String getWarna() {
        return warna;
    }

    public void setWarna(String warna) {
        this.warna = warna;
    }
}